src/modules/lib/FixedPointMathLib.sol

```diff
@@ -8,58 +8,56 @@ library FixedPointMathLib {
                             COMMON BASE UNITS
     //////////////////////////////////////////////////////////////*/
 
-    uint256 internal constant YAD = 1e8;
-    uint256 internal constant WAD = 1e18;
-    uint256 internal constant RAY = 1e27;
-    uint256 internal constant RAD = 1e45;
+    uint internal constant YAD = 1e8;
```

Comment:
Can you give a bit more context on these changes?

Reply:
Hmm I think that my formatter did that, I didn't go in and changed this manually. Do you think I should revert it so it's equal to the original Solmate version?

Reply:
I don't think it's worth it... was just curious.

Reply:
It is interesting to me though, as we now have an inconsistency here. Most of the codebase uses `uint256`, we now have some parts using `uint`. I know they are equivalent in terms of compilation, but I find it weird that the formatter seemed to convert some to `uint` and some not? Something isn't right here.
But we can also explore this in a different PR at a later time.